test/manual/maps/map_test.R

```diff
@@ -0,0 +1,193 @@
+box::use(gg = ggplot2)
+box::use(scales)
+box::use(gghdx)
+box::use(sf)
+box::use(dplyr)
+
+box::use(../../../src/images/plots/theme_signals)
+box::use(../../../src/images/maps/gg_map)
+box::use(../../../src/images/maps/geom_cities)
+box::use(../../../src/images/maps/geom_centroids)
+box::use(../../../src/images/maps/map_points)
+box::use(../../../src/utils/get_iso3_sf)
+box::use(cs = ../../../src/utils/cloud_storage)
+
+df_map_settings <- cs$read_az_file("input/iso3_map_settings.json")
+
+#' Create and save plots with randomly sampled values over admin zone of interest
+#' files will be saved in the directory specified with syntax: "{iso3}_map.png"
+#'
+#' This function is implemented for all iso3s in tests/manual/implement_test_maps.R
+#'
+#' @param iso3 ISO3 code
+#' @param sample_n `integer` vector (default = 1:20) sampled to randomly decide how many
+#'    pts will be mapped
+#' @param sample_values `integer` vector (defaults 1:20000) sampled to assign values to pts.
+#'    might be useful to have it sample a log distribution instead
+#' @param use_bbox `logical` if TRUE (default) samples will be drawn from the bounding box
+#'    of the polygon, if FALSE sampled withing the polygon. It's mainly a feature to
+#'    ease development as sampling withing bbox is much faster
+#' @param subtitle `character` subtitle to label plot
+#' @param map_settings json file containing map specifications (default = "input/iso3_map_settings.json")
+#' @param out_dir `character` directory to output sample maps. If NULL (default) no image written.
+#'    and no image written
+#'
+#' @return ggplot image
+#' @export
+
```

Comment:
This lints fine actually, but we can go with the standard of no space between roxygen and function in this repo?

Reply:
sure